src/com/owncloud/android/lib/resources/shares/UpdateRemoteShareOperation.java

```diff
@@ -102,6 +102,7 @@ public UpdateRemoteShareOperation(long remoteId) {
         mPassword = null;               // no update
         mExpirationDateInMillis = 0;    // no update
         mPublicUpload = null;
+        mPermissions = OCShare.DEFAULT_PERMISSION;
```

Comment:
Why default to this? Why not 0? Is there a way to not specifiy permissions in the operation, expecting that the current value is not changed?